conjure-java-jackson-serialization/src/main/java/com/palantir/conjure/java/serialization/ObjectMappers.java

```diff
@@ -97,6 +98,7 @@ public static ObjectMapper newCborServerObjectMapper() {
      */
     public static ObjectMapper withDefaultModules(ObjectMapper mapper) {
         return mapper
+                .setSerializationInclusion(JsonInclude.Include.NON_NULL)
```

Comment:
Instead we should probably use `NON_ABSENT` and remove the `configureAbsentsAsNulls` config below. This is what the docs recommend:
https://github.com/FasterXML/jackson-modules-java8/blob/jackson-modules-java8-2.9.9/datatypes/src/main/java/com/fasterxml/jackson/datatype/jdk8/Jdk8Module.java#L21-L23

Reply:
We should also probably just use `setDefaultPropertyInclusion(...)` instead. `setSerializationInclusion` is a just convenience method that calls `setPropertyInclusion` which is deprecated in favor of `setDefaultPropertyInclusion`.